meson_options.txt

```diff
@@ -18,6 +18,18 @@ option(
   description : 'Bash completions directory. Use "no" to disable.',
 )
 
+option(
+  'fishcompletiondir',
+  type : 'string',
+  description : 'Fish completions directory. Use "no" to disable.',
+)
+
+option(
+  'zshcompletiondir',
+  type : 'string',
+  description : 'Zsh completions directory. Use "no" to disable.',
+)
+
```

Comment:
next release will still include autotools. Shouldn't we also add these changes there?